frontend/src/metabase/hoc/Favicon.jsx

```diff
@@ -1,43 +1,14 @@
-import { useEffect } from "react";
+import { useFavicon as useFaviconMantineHook } from "@mantine/hooks";
```

Comment:
also probably worth migrating this to a `useFavicon` file outside of the hoc folder

Reply:
yes! definitely